src/optimize/base_optimizer.js

```diff
@@ -101,40 +135,24 @@ class BaseOptimizer {
 
       module: {
         loaders: [
-          {
-            test: /\.less$/,
-            loader: ExtractTextPlugin.extract(
-              'style',
-              `css${mapQ}!autoprefixer${mapQPre}{ "browsers": ["last 2 versions","> 5%"] }!less${mapQPre}dumpLineNumbers=comments`
-            )
-          },
-          {
-            test: /\.scss$/,
-            loader: ExtractTextPlugin.extract(
-              'style',
-              `css${mapQ}!autoprefixer${mapQPre}{ "browsers": ["last 2 versions","> 5%"] }!sass${mapQPre}`
-            )
-          },
-          { test: /\.css$/, loader: ExtractTextPlugin.extract('style', `css${mapQ}`) },
+          { test: /\.less$/, loader: makeStyleLoader('less') },
+          { test: /\.scss$/, loader: makeStyleLoader('sass') },
+          { test: /\.css$/, loader: makeStyleLoader() },
           { test: /\.jade$/, loader: 'jade' },
           { test: /\.json$/, loader: 'json' },
           { test: /\.(html|tmpl)$/, loader: 'raw' },
-          { test: /\.png$/, loader: 'url?limit=10000&name=[path][name].[ext]' },
-          { test: /\.(woff|woff2|ttf|eot|svg|ico)(\?|$)/, loader: 'file?name=[path][name].[ext]' },
-          { test: /[\/\\]src[\/\\](core_plugins|ui)[\/\\].+\.js$/, loader: `rjs-repack${mapQ}` },
+          { test: /\.png$/, loader: 'url' },
```

Comment:
It was effectively removed by #6177, and has been "working as intended" ever since, so rather than change the behavior again I decided to just match what we are currently doing. If we decided it's better to have then we can always add it back (preferably in another pr)